packages/flutter/lib/src/services/text_input.dart

```diff
@@ -814,6 +827,9 @@ class _TextInputClientHandler {
       case 'TextInputClient.updateFloatingCursor':
         _currentConnection._client.updateFloatingCursor(_toTextPoint(_toTextCursorAction(args[1]), args[2]));
         break;
+      case 'TextInputClient.onConnectionClosed':
```

Comment:
system_channels.dart documents the messages a channel uses. Can you please add that there?

Reply:
done. Thanks for the suggestion!